packages/features/form-builder/FormBuilder.tsx

```diff
@@ -509,6 +518,29 @@ function FieldEditDialog({
                         }}
                       />
                     ) : null}
+
+                    {fieldForm?.getValues("type") === "textarea" ? (
```

Comment:
I don't like the idea of checking field type here. There is a [dedicated fieldTypeConfig](https://github.com/calcom/cal.com/blob/a28eb63d611e956ad6d8fa03c2242fe8a0e8a712/packages/features/form-builder/fieldTypes.ts#L103) that should hold such features of a field type.

We could introduce something like `supportsLengthCheck` to textarea type field. 

It makes it easier to introduce a new type with same feature support.